osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/monitoring/GetPowerQualityProfileSelectiveAccessHandler.java

```diff
@@ -36,6 +36,9 @@ public GetPowerQualityProfileSelectiveAccessHandler(final DlmsHelper dlmsHelper)
         final List<ProfileEntryValueDto> result = new ArrayList<>();
         final List<DataObject> dataObjects = profileEntryDataObject.getValue();
 
+        System.out
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS106)